runtime/shiden/Cargo.toml

```diff
@@ -82,8 +82,8 @@ try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "p
 # Astar pallets
 pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-block-reward-2.0.0/polkadot-v0.9.17", default-features = false }
 pallet-custom-signatures = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.17", default-features = false }
-pallet-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-dapps-staking-3.2.0/polkadot-v0.9.17", default-features = false }
-pallet-precompile-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-dapps-staking-3.2.0/polkadot-v0.9.17", default-features = false }
+pallet-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", branch = "feature/compound-reward", default-features = false }
+pallet-precompile-dapps-staking = { git = "https://github.com/AstarNetwork/astar-frame", branch = "feature/compound-reward", default-features = false }
```

Comment:
Only Shibuya & local, Shiden must remain the same.
We will upgrade Shiden next week.

Reply:
ah.. I misunderstood. I thought because of the version bump the dependencies must be updated too.